fv3core/stencils/pe_halo.py

```diff
@@ -1,12 +1,17 @@
 from gt4py.gtscript import FORWARD, computation, horizontal, interval, region
 
-import fv3core._config as spec
-from fv3core.decorators import gtstencil
 from fv3core.utils.typing import FloatField
 
 
-@gtstencil()
 def edge_pe(pe: FloatField, delp: FloatField, ptop: float):
```

Comment:
Should this be moved into `dyn_core.py` and this file removed?

Reply:
I had a conversation with multiple people regarding this and the conclusion was that `dyn_core.py` is not the right place for this. 

We can see a future world where dyn_core contains ~5 objects, each with their own source files but I think this is the cleaner intermediate step

Reply:
Sounds good!